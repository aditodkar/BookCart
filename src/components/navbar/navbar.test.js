import React from 'react';
import renderer from 'react-test-renderer';
import Navbar from './navbar';

it('Navbar component snapshot', () => {
    const snap = renderer.create(<Navbar/>).toJSON();
    expect(snap).toMatchSnapshot();
});