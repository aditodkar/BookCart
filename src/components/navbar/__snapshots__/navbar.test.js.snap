// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar component snapshot 1`] = `
<nav
  className="navbar navbar-expand-lg navbar-dark primary-color"
>
  <a
    className="navbar-brand"
    href="#"
  >
    Books Cart
  </a>
  <div
    className="collapse navbar-collapse"
    id="navbarSupportedContent-4"
  >
    <ul
      className="navbar-nav ml-auto"
    >
      <li
        className="nav-item"
      >
        <p
          className="items"
        >
          Total Items: 
        </p>
      </li>
      <li
        className="nav-item"
      >
        <p
          className="productPrice"
        >
          Total Price: 
        </p>
      </li>
      <li
        className="nav-item"
      >
        <i
          className="fa fa-shopping-cart"
        />
      </li>
    </ul>
  </div>
</nav>
`;
